playbooks/robusta_playbooks/overcommit_enrichments.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from robusta.api import *
+
+
+@action
+def cpu_overcommited_enricher(alert: PrometheusKubernetesAlert, params: PrometheusParams):
+    """
+    Enrich the finding with a detailed explanation for the cause of the CPU overcommitment.
+    Includes recommendations for the identified cause.
+    """
+    cpu_analyzer = CpuAnalyzer(params.prometheus_url)
+    cpu_requests = cpu_analyzer.get_total_cpu_requests(timedelta(minutes=10))
```

Comment:
10 minutes sounds good to me, but I'm not 100% sure we won't have to change it.
Would it be better to add it as a parameter to the playbook, with default value of 10?